README.md

```diff
@@ -242,7 +242,17 @@ Folowing command delete the session
 ```
 ocm backplane session --delete <session-name>
 ```
+## Promotion/Release cycle of backplane CLI
+Backplane CLI has the following release cycle : 
 
+### Staging:
```

Comment:
I do not think we have staging for backplane-cli.

We can say the default release cycle would be 2 weeks for backplane-cli.